spec/support/common_actions.rb

```diff
@@ -357,4 +357,25 @@ def nested_path_for(resource)
     end
   end
 
+  def deliver_newsletter
```

Comment:
I don't see a point of making two helper functions (this one and fill_in_newsletter_form) that are only used once in the codebase... Instead I would avoid having 5 different calls to ` fill_in "newsletter_subject"` and all the other 4 lines of interaction with the form... and just have a "fill_newsletter_form" helper function that has default parameters and allows overwiting those that need to change (and also doesn't have expectations or other actions like `click_link "send"` that may not be expected when calling it on other places.